src/components/DateRangeInput.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Datepicker from 'react-tailwindcss-datepicker';
```

Comment:
new dependency. Sadly it's not as customisable as I would like but for now it offers the functionality we need, we can always contribute in the future.